packages/contracts-bedrock/snapshots/abi/FeeCurrency.json

```diff
@@ -0,0 +1,354 @@
+[
```

Comment:
Why are this and the file containing the slots added? Shouldn't they at least be called `AbstractFeeCurrency*`?

Reply:
These are for `testing/FeeCurrency.sol`, so the naming is fine. We don't need any for `AbstractFeeCurrency` as it can't be deployed anyway. I assume this file was missing before due to the name clash between the two contracts, but I didn't check how the snapshotting actually works.